packages/web-components/src/components/back-to-top/__stories__/data/content.js

```diff
@@ -0,0 +1,772 @@
+/* eslint-disable */
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+// TODO: replace with proper React components once react-wrapper webpack.config.js loaders are correctly configured
```

Comment:
Do you want to start with renaming this to `.tsx`? In the React wrapper Storybook environment, it's the only extension where Babel sees it as a React file.